pillar/top.sls

```diff
@@ -1,5 +1,6 @@
 base:
-  '* and not roles:proxy':
+  '* and not G@roles:proxy':
```

Comment:
There shouldn't be any need for the `* and`. Just having the `not` clause will implicitly include everything else.